.github/workflows/_build_azurefs_sas_token_provider_conda.yml

```
@@ -34,10 +31,12 @@ jobs:
       - name: 'Set Secret File Permissions and Conda Build and Publish Bodo-AzureFS-SAS-Token-Provider Binary to Artifactory'
         run: |
           set -eo pipefail
-          echo "${{ secrets.PUBLISH_BINARY_SECRETS }}" > $HOME/secret_file
-          sudo chmod a+r $HOME/secret_file
 
           artifactory_channel="bodo.ai"
-          echo "artifactory_channel: $artifactory_channel"
-          ./buildscripts/azurefs-sas-token-provider/publish_binary.sh $artifactory_channel
+          ./buildscripts/azurefs-sas-token-provider/publish_binary.sh "bodo.ai"
```

Shouldn't this use `BODO_CHANNEL_NAME`
```suggestion
          ./buildscripts/azurefs-sas-token-provider/publish_binary.sh $BODO_CHANNEL_NAME
```

---

I don't think it was before